acstools/acsphotcte.py

```diff
@@ -167,6 +167,9 @@ def _check_inputs(self, **inputs):
             it returns the corresponding key. If the iterable inputs are not
             all the same length, it returns `None`.
         """
+        # MJD date of Servicing Mission 4, 2009-05-11
+        # This is the same date used for the CTE Webtool
+        sm4 = 54962.0
```

Comment:
Why is this different from the one that Norman defined in `acs_destripe`?

https://github.com/spacetelescope/acstools/blob/4284f9575943ab1b520e2aa83d27ace4d97d1e9d/acstools/acs_destripe.py#L75

Reply:
Not sure, the value I use corresponds to the launch date for SM4. The value listed in `acs_destripe.py` is 5 days later on May 16 2009. The discrepancy shouldn't be any issue and so I am happy to conform to the date listed in that file.

Reply:
Would be nice if ACS Team can agree on a single value. Please see #118.